syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -158,6 +152,8 @@ def mul(self, self_, other):
             trunc_res = np.where(gate, neg_nums, pos_nums)
 
             return trunc_res
+        else:
+            return self_ + other
```

Comment:
This is a * not a + :)

Reply:
Eagle eye! :)
Also... quite concerned the tests are passing. I'm reviewing them

Reply:
Once `.float_precision()` is applied to the result it does not work. I'll keep working on it to fix it

Reply:
I'm postponing the implementation of `mul` when the tensors are shared. It will come in another PR as it is a bit tricky.